### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by BoxiLi**
Add a short introduction and copies support from qutip readme.
Add installation guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
 # qutip-qip
-The QuTiP quantum information processing package
+
+The qutip-qip package used to be a module ``qutip.qip`` under [QuTiP (Quantum Toolbox in Python)](http://qutip.org/index.html).
+From QuTiP 5.0, the community has decided to reduce the size of the core QuTiP package for reducing the external dependency and ease of maintenance.
```
Comment (nathanshammah) [inline README.md:4]:
```suggestion
From QuTiP 5.0, the community has decided to decrease the size of the core QuTiP package by reducing the external dependencies, in order to simplify maintenance.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+Hence a few modules are separated from the core QuTiP and become QuTiP family packages.
```
Comment (nathanshammah) [inline README.md:5]:
```suggestion
Hence a few modules are separated from the core QuTiP and will become QuTiP family packages.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+They are still maintained by the QuTiP team but hosted under different repositories in the [QuTiP organization](https://github.com/qutip).
+
+The qutip-qip package, QuTiP quantum information processing, aims at providing basic tools for quantum computing simulation both for simple quantum algorithm design and for experimental realization.
+Compared to other libraries for quantum information processing, qutip-qip put additional emphasis on the physics layer and the interaction with the QuTiP package.
```
Comment (nathanshammah) [inline README.md:9]:
```suggestion
Compared to other libraries for quantum information processing, qutip-qip puts additional emphasis on the physics layer and the interaction with the QuTiP package.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+under the directory containing the ``setup.py`` file.
+
+If you want to edit the code, use instead
+```
+    pip install -e .
+```
+
+Support
+-------
+This package is supported and maintained by the same group as QuTiP.
```
Comment (nathanshammah) [inline README.md:27]:
```suggestion
This package is supported and maintained by the same developers group as QuTiP.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,41 @@
...
+
+[![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org)
+[![Unitary Fund](https://img.shields.io/badge/Supported%20By-UNITARY%20FUND-brightgreen.svg?style=for-the-badge)](http://unitary.fund)
+
+
+QuTiP development is supported by [Nori's lab](http://dml.riken.jp/)
+at RIKEN, by the University of Sherbrooke, and by Aberystwyth University,
```
Comment (nathanshammah) [inline README.md:34]:
```suggestion
at RIKEN, by the University of Sherbrooke, by Chalmers University of Technology, by Macquarie University and by Aberystwyth University,
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Small changes suggested, thanks @BoxiLi.

**Comment by BoxiLi** [comment]
All suggestions applied. Thanks @nathanshammah

**Pull request merged**